app/gist/new/route.js

```diff
@@ -7,7 +7,7 @@ export default GistRoute.extend({
   model () {
     this.store.unloadAll('gistFile');
 
-    var model = this.store.createRecord('gist', {description: 'New Twiddle'});
+    var model = this.store.createRecord('gist', {description: 'New Twiddle '});
```

Comment:
So you have to

1. Change the title
2. Save the gist
3. Load Ember Twiddle with the saved gist.